src/main/scala/wdl4s/values/WdlFile.scala

```diff
@@ -35,14 +39,20 @@ sealed trait WdlFile extends WdlPrimitive {
     case _ => invalid(s"$value == $rhs")
   }
 
-  override def valueString = value.toString
+  override def valueString: String = value.toString
 }
 
-case class WdlSingleFile(value: String) extends WdlFile {
-  override def toWdlString = "\"" + value.toString + "\""
+case class WdlSingleFile(origValue: String) extends WdlFile {
+  override def toWdlString: String = "\"" + value.toString + "\""
+
+  // Validate file path
```

Comment:
If I understood @jsotobroad correctly, the URI type contains a whole bunch of baggage and difficulty in interpretation. 

I liked the idea of adding the `Refined` type since it lets us check that the string is valid, but when we want to use it, it's still just a String object and is just as easy to manipulate.